graphql/schemabuilder/pagination.go

```diff
@@ -422,15 +422,15 @@ func (c *connectionContext) applyTextFilter(nodes []interface{}, args Pagination
 		// For each possible field we're matching against, check if there's a match.
 		for _, filterField := range c.FilterTextFields {
 			// Resolve the graphql.Field made for sorting.
-			text, err := filterField.Resolve(context.TODO(), node, nil, nil)
```

Comment:
This context ensures we keep live data / caching going

Reply:
oh great please ignore my earlier comments